CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 4.5.0-beta.0 (July 25, 2019)
+===================================
+
+ * [New](https://github.com/segmentio/analytics-android/pull/620): Add Application Open and Application Backgrounded Events
```

Comment:
I think the PR title was a bit misleading - but the change was a) we added Application Backgrounded events and b) we now report Application Open events from background launches (we only used to do it from a cold launch previously).